src/test/java/teammates/test/cases/util/SanitizationHelperTest.java

```diff
@@ -95,6 +97,58 @@ private void sanitizeHtml_receivesSanitized_returnsUnchanged() {
         assertEquals(sanitized, SanitizationHelper.sanitizeForHtml(sanitized));
     }
 
+    @Test
+    public void testSanitizeForHtmlList() {
+        List<String> input = new ArrayList<>();
+        input.add("apple <");
+        input.add("banana ' dogs ");
+        input.add("rollercoasters & tycoons");
+        input.add("");
+        input.add(null);
+
+        List<String> expected = new ArrayList<>();
+        expected.add("apple &lt;");
+        expected.add("banana &#39; dogs ");
+        expected.add("rollercoasters &amp; tycoons");
+        expected.add("");
+        expected.add(null);
+
+        // standard case
+        assertEquals(expected, SanitizationHelper.sanitizeForHtml(input));
+
+        // empty input list
+        assertEquals(new ArrayList<String>(), SanitizationHelper.sanitizeForHtml(new ArrayList<>()));
```

Comment:
Something like this would be self-documenting:
https://github.com/TEAMMATES/teammates/blob/9811580aad4bcfbe670a0ced33f9ff92057b116c/src/test/java/teammates/test/cases/util/SanitizationHelperTest.java#L255-L256

Reply:
Do you think this will cause an issue due to side effects? e.g. what if the method will cause a modification of `emptyList` such as through a bug or regression. Then the expected output (which is the same object) also changes to something that is incorrect, causing the test to pass incorrectly.

Reply:
That's true 👍 It's fine then.